code/game/machinery/cryo.dm

```diff
@@ -122,7 +122,14 @@
 	..()
 	if(autoeject)
 		if(occupant)
-			if(occupant.health >= 100)
+			var/mech_damage = 0
+			if(istype(occupant, /mob/living/carbon/human))

```

Comment:
`if(ishuman(occupant))`

Don't use typechecks when istype helpers exist; they're slower and less maintainable.